crowdsourcing/models.py

```diff
@@ -274,7 +274,7 @@ class TemplateItem(models.Model):
     data_source = models.CharField(max_length=256, null=True)
     layout = models.CharField(max_length=16, default='column')
     type = models.CharField(max_length=16)
-    sub_type = models.CharField(max_length=16)
```

Comment:
we map 1:1 to html so when I say I want a number input field I cannot specify the datatype other than number, or email or text otherwise the logic becomes more complicated unnecessarily but this comes later no worries